akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMap.scala

```diff
@@ -100,8 +151,17 @@ final class ORMap[A, B <: ReplicatedData] private[akka] (
         "`ORMap.put` must not be used to replace an existing `ORSet` " +
           "value, because important history can be lost when replacing the `ORSet` and " +
           "undesired effects of merging will occur. Use `ORMultiMap` or `ORMap.updated` instead.")
-    else
```

Comment:
I would keep this comment as is, because, actually, only `ORMultiMapWithDeltaDelta` is free from the anomalies here